extra/resources/ClusterMon

```diff
@@ -153,7 +153,8 @@ ClusterMon_monitor() {
     if [ -f $OCF_RESKEY_pidfile ]; then
 	pid=`cat $OCF_RESKEY_pidfile`
 	if [ ! -z $pid ]; then
-	    kill -s 0 $pid >/dev/null 2>&1; rc=$?
+	    ps -wwfp $pid | grep $OCF_RESKEY_pidfile >/dev/null 2>&1 && \
+	        kill -s 0 $pid >/dev/null 2>&1; rc=$?
```

Comment:
As otherwise pretty funny thing may happen against all odds: grep command will accept itself. So is that the main reason `kill` goes in a second line? :-)